dashboard/src/constants.ts

```diff
@@ -45,7 +47,7 @@ export const latestSchemaVersions = (schemaType: string, versionStr: string) =>
       return true;
     }
   } else if (schema === "generic") {
-    if (["2.1.1"].includes(versionStr)) {
+    if (["3.0.0"].includes(versionStr)) {
```

Comment:
Does this eliminate support for the older version?

Reply:
No, it just helps indicate whenever the subgraph does not have the latest schema version